src/lib/pages/account-details/index.tsx

```diff
@@ -142,10 +142,7 @@ const AccountDetailsBody = ({ accountAddress }: AccountDetailsBodyProps) => {
         <TabPanels>
           <TabPanel p={0}>
             <TotalAccountValue accountAddress={accountAddress} />
-            <Flex
-              borderBottom={{ base: "0px", md: "1px solid" }}
-              borderBottomColor={{ base: "transparent", md: "gray.700" }}
-            >
+            <Flex borderBottom="1px solid" borderBottomColor="gray.700">
```

Comment:
What about the mobile version? Is it still functioning properly?

Reply:
Yep, works perfectly splendid